Testscripts/Linux/verify_vmbus_heartbeat_properties.sh

```diff
@@ -55,18 +55,27 @@ SetTestStateRunning
 test_failed=""
 
 # check the file structure exists and that all the files can be read
-vmbus_driver_files=(channel_vp_mapping class_id client_monitor_conn_id \
+vmbus_driver_files_1=(channel_vp_mapping class_id client_monitor_conn_id \
 client_monitor_latency client_monitor_pending device device_id id in_intr_mask \
 in_read_bytes_avail in_read_index in_write_bytes_avail in_write_index modalias \
 monitor_id out_intr_mask out_read_bytes_avail out_read_index out_write_bytes_avail \
 out_write_index server_monitor_conn_id server_monitor_latency \
 server_monitor_pending state uevent vendor)
 
+vmbus_driver_files_2=(channel_vp_mapping class_id device device_id id \
+in_intr_mask in_read_bytes_avail in_read_index in_write_bytes_avail \
+in_write_index modalias out_intr_mask out_read_bytes_avail out_read_index \
+out_write_bytes_avail out_write_index state uevent vendor driver_override)
```

Comment:
The only differences are 'driver_override' and 'state'. Why don't you append those 2 if $sys_path/driver_override exists? Wonder if we keep the simple variable management.

Reply:
Seems it's not a simple super-set.  
'client_monitor_conn_id \
client_monitor_latency client_monitor_pending

Reply:
@LiliDeng I am not asking if this is really superset or not, but want to make the coding flow simple. Can you keep a single variable here and append those values if "$sys_path/driver_override" does not exist?